### DISCUSSION TITLE
nucle wallet

### DISCUSSION THREAD

**Discussion by Roman00000000** [Chia-Network/chia-blockchain#13420] [Support]
hi ,can you help my with my nucle wallet, i have a problem with it

**Comment by Iamalexoper** [comment]
A have the same problem, and many people have encountered this problem, wallet doent work, we ask the chia team to help us!

**Comment by Roman00000000** [comment]
Ср, 14 сент. 2022 г. в 08:22, Iamalexoper ***@***.***>:

> A have the same problem, and many people have encountered this problem,
> wallet doent work, we ask the chia team to help us!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Chia-Network/chia-blockchain/discussions/13420#discussioncomment-3641116>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3ABQQDUMPN2XN4SNNEGTD3V6FOKBANCNFSM6AAAAAAQJUCZ6M>
> .
> You are receiving this because you authored the thread.Message ID:
> <Chia-Network/chia-blockchain/repo-discussions/13420/comments/3641116@
> github.com>
>
thank you, you can keep me informed, I will be very grateful

**Reply by WPPace** [reply]
Hello, I have the same problem. For more than a week I have 0 xch in my wallet
